README.md

```diff
@@ -1,37 +1,103 @@
-# MarkLogic Unit Test
+![GitHub release](https://img.shields.io/github/release/marklogic-community/marklogic-unit-test.svg)
+![GitHub last commit](https://img.shields.io/github/last-commit/marklogic-community/marklogic-unit-test.svg)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+# Testing MarkLogic
 
-marklogic-unit-test includes the [original Roxy unit test framework for MarkLogic](https://github.com/marklogic-community/roxy/wiki/Unit-Testing) and 
-provides a few new features:
+marklogic-unit-test is an [ml-gradle bundle](https://github.com/marklogic-community/ml-gradle/wiki/Bundles) that allows
+a project to test MarkLogic code.  With one import a project immediately has access to:
 
-1. A REST endpoint for listing and running unit tests
-1. A small Java library for integrating MarkLogic unit tests into existing test frameworks like JUnit
-1. The ability to depend on this module as a true third-party dependency via ml-gradle
+1. A framework for writing and running MarkLogic unit tests, including several built in assertion functions
+1. A UI for viewing and running unit tests entirely within MarkLogic
+1. A REST endpoint to run and report unit tests with other tools
 
-To try this out locally, check out the [ml-gradle example project](https://github.com/marklogic-community/ml-gradle/tree/dev/examples/unit-test-project). 
+Testing MarkLogic from a Java project is made easy with marklogic-junit:
+
+1. Write MarkLogic tests entirely from Java
+1. Easily integrate MarkLogic unit tests into your favorite Java testing frameworks
+
+# Start using marklogic-unit-test
+
+If you'd like to use marklogic-unit-test check out this 
+[ml-gradle example project](https://github.com/marklogic-community/ml-gradle/tree/dev/examples/unit-test-project). 
 You can use that project's build.gradle file as an example of how to use marklogic-unit-test in your own project.
 
-## Releasing
+### Add marklogic-unit-test to `build.gradle`
+
```

Comment:
I'm thinking this should be on a Wiki page now. I say that because of the point I made above - I think because marklogic-unit-test has multiple user interfaces to it, we should highlight those ASAP instead of showing code-y stuff (as opposed to ml-gradle or ml-app-deployer, which don't have any interfaces besides a command line). 

This detail is important, but I'm thinking it's on the Wiki home page. 

So after the example test / screenshots on the home page, toss a big link in for "Here's how to get started using marklogic-unit-test".